NEWS

```diff
@@ -98,8 +101,9 @@ Bug fixes/minor improvements:
     "file:filename").
   - orte_timeout_for_stack_trace: number of seconds to wait for stack
     traces to be reported (or <=0 to wait forever).
-- Various improvements to the Portals 4 MTL, to include adding support
-  for non-contiguous datatypes.
+  - mtl_ofi_control_prog_type/mtl_ofi_data_prog_type: specify libfabric
+    progress model to be used for control and data
+- Add support for non-contiguous datatypes to the Portals 4 OSC.
```

Comment:
Should probably say "one-sided component" here (I don't think "OSC" is a well-known acronym outside of the developer community).